addon/HomepageStudy.jsm

```diff
@@ -74,16 +74,25 @@ function getHomepage() {
 var HomepageStudy = {
   /**
    * Returns the value encoded by RAPPOR or null if the homepage can't be obtained.
+   * The two last arguments are not used, but must be declared for consistence with Simulator.jsm
    * @param {string} studyName - Name of the study.
+   * @param {boolean} isSimulation - Boolean indicating whether the execution is for a simulation.
```

Comment:
Drop the docs about the unused arguments.

Reply:
Done